mkt/api/middleware.py

```diff
@@ -262,17 +262,27 @@ def process_response(self, request, response):
             getattr(request, 'CORS_HEADERS',
                     ('X-HTTP-Method-Override', 'Content-Type')))
 
-        if fireplacey or getattr(request, 'CORS', None):
+        error_allowed_methods = []
+        # If settings.DEBUG is True, these skip the 404 and 500 handlers
```

Comment:
Ah right, had not seen that. Aren't middlewares also run anyway when we go through `handler_*` ? If so, any reason to do the job in different places instead of consolidating everything in the middleware ?